### PR TITLE
Merge 1.7.7.x into develop - 30/01/2020

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merge last changes from 1.7.7.x into develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17442)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js [hunk 1]
```diff
@@ -52,6 +52,12 @@ export default class CurrencyForm {
     this.$resetDefaultSettingsButton = $(this.map.resetDefaultSettingsInput);
     this.$loadingDataModal = $(this.map.loadingDataModal);
     this.currencyFormatterId = this.map.currencyFormatter.replace('#', '');
+    this.hideModal = true;
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js:55]: Weird 🤔 maybe a git conflict resolution gone wrong

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js [hunk 2]
```diff
@@ -77,9 +77,9 @@ export default class OrderProductRenderer {
   }
 
   moveProductsPanelToRefundPosition() {
-    this.moveProductPanelToTop();
     /* eslint-disable-next-line max-len */
     $(`${OrderViewPageMap.productAddActionBtn}, ${OrderViewPageMap.productAddRow}, ${OrderViewPageMap.productActionBtn}`).addClass('d-none');
+    this.moveProductPanelToTop();
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js:82]: Need to confirm if this is not an issue

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js [hunk 3]
```diff
@@ -194,9 +200,26 @@ export default class OrderProductRenderer {
   }
 
   toggleProductAddNewInvoiceInfo() {
-    $(OrderViewPageMap.productAddNewInvoiceInfo).toggleClass(
-      'd-none',
-      $(OrderViewPageMap.productAddInvoiceSelect).val() === 0,
-    );
+    if ($(OrderViewPageMap.productAddInvoiceSelect).val() === 0) {
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js:203]: Need to confirm it's correct (replace `toggleClass()` by `removeClass('d-none')` / `addClass('d-none')` )

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js:203]: use the `toggleClass` instead

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js (hunk 2 above)
Comment (jolelievre) [reply admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js:82]: Seems good to me

File: admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js (hunk 1 above)
Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js:55]: LGTM

**Comment by matks** [comment]
Closed in favor of https://github.com/PrestaShop/PrestaShop/pull/17466/

**Pull request closed without merging**